drivers/flash/soc_flash_rv32m1.c

```diff
@@ -142,6 +142,9 @@ static const struct flash_driver_api flash_mcux_api = {
 	.page_layout = flash_mcux_pages_layout,
 #endif
 	.write_block_size = FSL_FEATURE_FLASH_PFLASH_BLOCK_WRITE_UNIT_SIZE,
+#if IS_ENABLED(CONFIG_FLASH_ERASE_VALUE)
```

Comment:
For all these: if people accept that my argument that the value needs to support a runtime determination then all these need to move to the device-specific flash or ram structure.